pkg/kubelet/cm/cgroup_manager_linux.go

```diff
@@ -449,8 +449,7 @@ func (m *cgroupManagerImpl) Update(cgroupConfig *CgroupConfig) error {
 	}()
 
 	// Extract the cgroup resource parameters
-	resourceConfig := cgroupConfig.ResourceParameters
-	resources := m.toResources(resourceConfig)
+	resources := m.toResources(cgroupConfig.ResourceParameters)
```

Comment:
Again, this is a separate commit.

I'm not very familiar with kubernetes processes. Isn't it possible to revert separate commits, rather than the whole PR?